redhat-developer/app-services-cli
# Connector in context

This pr adds a connector to the context used in the cli. With this a new command is added `rhoas connector use` which is also aliased in `rhoas context set-connector`. 

### Verification Steps
Start by deleting the current context and creating a new one, also and create a connector instance in the UI.

1. Run `rhoas context set-connector` to get a list of your connectors and set one
2. Run `rhoas context status` to see the status of that connector.
3. Run `cat /home/$USER/.config/rhoas/contexts.json` to see saved context file

### Type of change
